src/HostManager.cpp

```diff
@@ -40,10 +40,8 @@
 
 bool HostManager::deleteHost(QString hostname)
 {
-    qDebug() << "HostManager::deleteHost(" << hostname.toUtf8().constData() << ") INFO: trying to delete host from host pool, getting write lock...";
```

Comment:
👍 Yeah these were meant to be got rid-of,  I just was running short of _roundtuits_.